### ISSUE TITLE
Possible issue in find_default_msbuild_path function from MSProgram-Get.psm1

### ISSUE THREAD

**Issue by omeuid** [postgresql-interfaces/psqlodbc#29]
I understand that the find_default_msbuild_path function from the MSProgram-Get.psm1 script could be wrong.

I would change the condition from the following IF statement:

```
    if ($env:PROCESSOR_ARCHITECTURE -eq "x86" -or $env:PROCESSOR_ARCHITECTURE -eq "ARM64") {
        $pgmfs = "$env:ProgramFiles"
    } else {
        $pgmfs = "${env:ProgramFiles(x86)}"
    }
```

As the objective seems to be the correct selection of the program files directory based on the processor architecture I would change the first line with the following one:
```
if ($env:PROCESSOR_ARCHITECTURE -eq "x64" -or $env:PROCESSOR_ARCHITECTURE -eq "ARM64") {
```

Am I missing something or should that change be done?

Regards,
Carlos

**Comment by apgrucza** [comment]
@omeuid the code shown above is out of date. The problem you are experiencing was likely fixed in #9.

**Comment by omeuid** [comment]
Hi,

Thanks for the quick response. I was building the current release, so I was in tag `REL-16_00_0005` where the problem is reproduced. 

I will close this ticket.
